src/middleware/authMiddleware.js

```diff
@@ -45,6 +39,10 @@ export function login(req, res) {
  */
 
 export default async function authMiddleware(req, res, next) {
+  // bypass authorization, used for cucumber UAT and axe accessibility testing
+  if (process.env.NODE_ENV !== 'production' && process.env.BYPASS_AUTH === 'true') {
```

Comment:
Glad to see the additional `NODE_ENV` check 👍🏻

Reply:
I'm nothing if not paranoid :)